### PR TITLE
Updated UserInterface and build.gradle.kts

### PR DESCRIPTION
**Pull request by OakLoaf**

### CODE REVIEW

**Comment by OakLoaf** [comment]
The configurable aliases that I just added, seem to cause some issues when it comes to unregistering - will look into it in the morning

**Comment by OakLoaf** [comment]
This should all be resolved now :)

File: build.gradle.kts [hunk 1]
```diff
@@ -16,34 +15,47 @@ repositories {
...
+    implementation("net.kyori:adventure-platform-bukkit:4.1.1")
+    implementation("mysql:mysql-connector-java:8.0.29")
+    implementation("com.zaxxer:HikariCP:5.0.1")
+    implementation("org.xerial:sqlite-jdbc:3.36.0.3")
+    implementation("com.github.Xemorr:ConfigurationData:537df1d205")
+    implementation("me.xemor:UserInterface:2.0.1-SNAPSHOT")
 }
 
 java {
-    configurations.compile.get().dependencies.remove(dependencies.gradleApi())
```
Comment (Xemorr) [inline build.gradle.kts:36]: Why did you remove this line out of interest? It was originally added because the entire gradle api was being shaded into plugins?

Comment (Xemorr) [reply build.gradle.kts:36]: or have you double checked that yours doesn't do this

Comment (OakLoaf) [reply build.gradle.kts:36]: I used to have it in all my plugins and then stumbled into some issues at one point, iirc kotlin-dsl was what was causing the gradle api to be shaded in

Comment (Xemorr) [reply build.gradle.kts:36]: Could you whip out WinRAR / 7zip and see if it's being compiled into this?

Comment (OakLoaf) [reply build.gradle.kts:36]: Comparing PlayerShopOverhaul to ActivityRewarder, I presume all the additional packages are drivers and stuff for adventure? No sign of the gradle packages though
image

Comment (Xemorr) [reply build.gradle.kts:36]: Great to hear, you're probably right.
